src/main/java/org/isf/admission/service/AdmissionIoOperationRepositoryImpl.java

```diff
@@ -39,23 +39,22 @@
 @Transactional
 public class AdmissionIoOperationRepositoryImpl implements AdmissionIoOperationRepositoryCustom {
 
-	private static String nativeQueryTerms = "SELECT * from OH_patient as p  "
-			+ " left join (select * from OH_admission where ADM_IN = 1 and ( (ADM_DELETED='N') or (ADM_DELETED is null ) ) ) as a on p.PAT_ID = a.ADM_PAT_ID "
-			+ " where ( ( p.PAT_DELETED='N' ) or ( p.PAT_DELETED is null ) )"
-			+ " and ( lower(concat_ws(' ', p.PAT_ID, p.PAT_SNAME, p.PAT_FNAME, p.PAT_NAME, p.PAT_NOTE, p.PAT_TAXCODE, p.PAT_CITY, p.PAT_ADDR, p.PAT_TELE)) like :param0 ) "
-			+ " order by p.PAT_ID desc";
-
-	private static String nativeQueryRanges = "SELECT * from OH_patient as p  "
-			+ " left join (select * from OH_admission where ADM_IN = 1 and ( (ADM_DELETED='N') or (ADM_DELETED is null ) ) ) as a on p.PAT_ID = a.ADM_PAT_ID "
-			+ " where (p.PAT_ID IN (SELECT ADM_PAT_ID from OH_admission where param1))"
-			+ " and ( lower(concat_ws(' ', p.PAT_ID, p.PAT_SNAME, p.PAT_FNAME, p.PAT_NAME, p.PAT_NOTE, p.PAT_TAXCODE, p.PAT_CITY, p.PAT_ADDR, p.PAT_TELE)) like :param0 ) "
-			+ " order by p.PAT_ID desc";
-
-	private static String nativeQueryCode = "SELECT * from OH_patient as p  "
-			+ " left join (select * from OH_admission where ADM_IN = 1 and ( (ADM_DELETED='N') or (ADM_DELETED is null ) ) order by ADM_ID desc) as a on p.PAT_ID = a.ADM_PAT_ID "
-			+ " where p.PAT_ID = :param0 "
-			+ " and ( ( p.PAT_DELETED='N' ) or ( p.PAT_DELETED is null ) )";
-
+	private static String nativeQueryTerms = "SELECT * from oh_patient as p  "
+					+ " left join (select * from oh_admission where ADM_IN=1 and ( ADM_ACTIVE=1 ) ) as a on p.PAT_ID=a.ADM_PAT_ID "
+					+ " where p.PAT_ACTIVE=1 "
+					+ " and ( lower(concat_ws(' ', p.PAT_ID, p.PAT_SNAME, p.PAT_FNAME, p.PAT_NAME, p.PAT_NOTE, p.PAT_TAXCODE, p.PAT_CITY, p.PAT_ADDR, p.PAT_TELE)) like :param0 ) "
+					+ " order by p.PAT_ID desc";
+
+			private static String nativeQueryRanges = "SELECT * from oh_patient as p  "
```

Comment:
Why is this String and the next String indented so far into the file and not aligned with the String field in line 42?

Reply:
Fixed, thanks!